admin/cluster_setup.py

```diff
@@ -205,6 +208,9 @@ def main(reactor, args, base_path, top_level):
                                )
         gather_deferreds(results)
 
+        flocker_client = _make_client(reactor, cluster)
+        yield _wait_for_nodes(flocker_client, len(cluster.agent_nodes))
```

Comment:
But since you already have the `reactor` local here, perhaps better than renaming `flocker_client._reactor` to `flocker_client.reactor` would be to have `_wait_for_nodes` accept the reactor as a separate parameter and pass it in explicitly at this call site.